sdk/attestation/azure-security-attestation/test/samples/basic-operations/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ add_executable (
 
 create_per_service_target_build(attestation-basic-operations attestation)
 
-target_link_libraries(attestation-basic-operations PRIVATE azure-security-attestation azure-identity get-env-helper)
+target_link_libraries(attestation-basic-operations PRIVATE azure-core azure-security-attestation azure-identity get-env-helper)
```

Comment:
interesting. We should not require customers to link azure-core.  azure-core should be a dependency for azure-security-attestation and should be linked by attestation.  This does not look right if we need to link azure-core.

Reply:
When I tried building Attestation directly, I had compilation failures without adding azure-core directly. I'll pull it out for now, but we're likely to have to revisit this when we do the work to ensure that every individual package builds without building from the root.

Reply:
it makes sense to see `azure-attestation` linking `azure-core`.  The package library requires and must link azure-core.
Then, any other library or binary linking `azure-attestation` should not need to link `azure-core`